lms/djangoapps/support/static/support/templates/enrollment-modal.underscore

```diff
@@ -0,0 +1,27 @@
+<div class="enrollment-modal">
+  <h1 class="enrollment-change-title">Change enrollment for <%- enrollment.get('course_id') %></h1>
+  <p class="enrollment-change-errors" style="display: none;"></p>
+  <ol class="enrollment-info">
```

Comment:
Why is the content being rendered as a list? Screenreaders are very verbose about entering lists which leads to poor user experience. I would recommend moving to divs.